docs/src/examples/ITensor.md

```diff
@@ -0,0 +1,70 @@
+# ITensor Code Examples
+
+
+## Print Indices of an ITensor
+
+Sometimes the printout of an ITensor can be rather large, whereas you
+might only want to see its indices. For these cases, just wrap the
+ITensor in the function `inds` like this:
+
+```julia
+@show inds(T)
+```
+
+or this
+
+```julia
+println("T inds = ",inds(T))
+```
+
+
+## Write and Read an ITensor to Disk with HDF5
+
+Saving ITensors to disk can be very useful. For example, you
+might encounter a bug in your own code, and by reading the
+ITensors involved from disk you can shortcut the process of
+running a lengthy algorithm over many times to reproduce the bug.
+Or you can save the output of an expensive calculation, such as
+a DMRG calculation, and use it as a starting point for multiple
+follow-up calculations such as computing time-dependent properties.
+
+ITensors can be written to files using the HDF5 format. HDF5 offers
+many benefits such as being portable across different machine types,
+and offers a standard interface across various libraries and languages.
+
+**Writing an ITensor to an HDF5 File**
+
+Let's say you have an ITensor `T` which you have made or obtained
+from a calculation. To write it to an HDF5 file named "myfile.h5"
+you can use the following pattern:
+
+```julia
+using HDF5
```

Comment:
This would require that users `add HDF5` themselves. We could instead use `using ITensors.HDF5` here which doesn't require that extra step.

Reply:
That's a nice observation - I didn't know that, and was just expecting people to add HDF5, yes.

Do you think if we changed it to `using ITensors.HDF5` that people might think that's a different package than HDF5.jl or be confused by it for some other reason?

Reply:
Hmm, I don't know. We could just make a comment about what that notation means, and say it is equivalent to `using HDF5` if you `add HDF5`.